logstash-core-event-java/src/main/java/com/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -1,15 +1,10 @@
 package com.logstash.ext;
 
-import com.logstash.Event;
-import com.logstash.PathCache;
-import com.logstash.RubyToJavaConverter;
-import com.logstash.Timestamp;
+import com.logstash.*;
```

Comment:
it is possible to configure intellij to not do this. I think you have to set a threshold class count before wildcarding (and just set it to something crazy).